### PR TITLE
convert to https://www.apache.org/licenses/LICENSE-2.0

### PR DESCRIPTION
**Pull request by Christian-B**
all links to apache license converted to https
including validation example

If approved will do the same in other repositories

### CODE REVIEW

File: SpiNNaker-front-end/src/test/resources/uk/ac/manchester/spinnaker/front_end/iobuf/empty.aplx [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# Copyright (c) 2022 The University of Manchester
```
Comment (dkfellows) [inline SpiNNaker-front-end/src/test/resources/uk/ac/manchester/spinnaker/front_end/iobuf/empty.aplx:1]: Looks like this file is having an EOL fight...

File: SpiNNaker-allocserv/src/main/resources/queries/find_rectangle_at.sql [hunk 2]
```diff
@@ -49,4 +49,4 @@ SELECT
 FROM args, selected_root, root_count
 WHERE root_count.available >= args.width * args.height - args.max_dead_boards
 	AND selected_root.may_be_allocated > 0
-LIMIT 1;
```
Comment (dkfellows) [inline SpiNNaker-allocserv/src/main/resources/queries/find_rectangle_at.sql:52]: Curious. This file was missing a terminating newline. (I approve of adding it.)

File: SpiNNaker-allocserv/src/main/resources/queries/connected_boards_at_coords.sql [hunk 3]
```diff
@@ -23,7 +23,7 @@ WITH RECURSIVE
 				FROM machines JOIN args USING (machine_id)
 				LIMIT 1),
 			xrange(n) AS (
-				SELECT 0 UNION ALL SELECT n+1 FROM xrange, args 
+				SELECT 0 UNION ALL SELECT n+1 FROM xrange, args
```
Comment (dkfellows) [inline SpiNNaker-allocserv/src/main/resources/queries/connected_boards_at_coords.sql:26]: I approve of the remoal of this trailing space.

File: SpiNNaker-allocserv/src/main/resources/application.yml [hunk 4]
```diff
@@ -64,7 +64,7 @@ server:
 
 cxf:
   path: /spalloc/srv
-  servlet.init: 
+  servlet.init:
```
Comment (dkfellows) [inline SpiNNaker-allocserv/src/main/resources/application.yml:67]: I approve of the removal of the trailing spaces from hereon in this file.

File: .github/workflows/publish.yml [hunk 5]
```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Build
         run: mvn install --settings $SETTINGS_FILE -DskipTests=true -Dmaven.javadoc.skip=true
-        
+
```
Comment (dkfellows) [inline .github/workflows/publish.yml:51]: I approve of the removal of this trailing space.

File: .github/workflows/publish-pkgs.yml [hunk 6]
```diff
@@ -18,10 +18,10 @@ on:
     types: [created]
 jobs:
   publish:
-    runs-on: ubuntu-latest 
-    permissions: 
+    runs-on: ubuntu-latest
```
Comment (dkfellows) [inline .github/workflows/publish-pkgs.yml:21]: I approve of the removal of these trailing spaces.

File: .github/workflows/build.yml [hunk 7]
```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 name: Build and Test
-on: 
+on:
```
Comment (dkfellows) [inline .github/workflows/build.yml:18]: I approve of the removal of this trailing space.

**Review by dkfellows — APPROVED** [review]
Few places have unexpected cleanup, but as long as `empty.aplx` is being converted to use LF instead of CRLF, I approve. (I probably missed doing it because I have APLX files marked as binary and skipped all binary files during my cleanup PR.)

File: SpiNNaker-front-end/src/test/resources/uk/ac/manchester/spinnaker/front_end/iobuf/empty.aplx (hunk 1 above)
Comment (Christian-B) [reply SpiNNaker-front-end/src/test/resources/uk/ac/manchester/spinnaker/front_end/iobuf/empty.aplx:1]: pycharm magic.
I used ubuntu file to check and yes in master it says "ASCII text, with CRLF line terminators" while the branch says "ASCII text"

**Pull request merged**